amazon-chime-sdk/src/main/java/com/amazonaws/services/chime/sdk/meetings/internal/audio/DefaultAudioClientController.kt

```diff
@@ -49,23 +51,34 @@ class DefaultAudioClientController(
         var audioClientState = AudioClientState.INITIALIZED
     }
 
-    private fun setUpAudioConfiguration() {
+    private fun setUpAudioConfiguration(audioMode: AudioMode) {
         // There seems to be no call that gives us the native input sample rate, so we just use the output rate
         val nativeSR = AudioTrack.getNativeOutputSampleRate(AudioManager.STREAM_SYSTEM)
 
         // The HARDWARE_SAMPLERATE is currently used to construct the proper buffer sizes
         audioClient.sendMessage(AudioClient.MESS_SET_HARDWARE_SAMPLE_RATE, nativeSR)
 
         // This IO_SAMPLE_RATE is used to create OpenSLES:
+        val samplingRateConfig = when (audioMode) {
+            AudioMode.Mono16K -> 16000
+            AudioMode.Mono48K -> 48000
+            AudioMode.Stereo48K -> 48000
+        }
         audioClient.sendMessage(
             AudioClient.MESS_SET_IO_SAMPLE_RATE,
-            AudioClient.AUDIO_CLIENT_SAMPLE_RATE
+            samplingRateConfig
         )
 
         // Result is in bytes, so we divide by 2 (16-bit samples)
+        // Result is in bytes, so we divide by 2 (16-bit samples)
+        val channelConfig = when (audioMode) {
```

Comment:
`outputChannelConfig` would be more descriptive.

Reply:
Modified to use this name.